tests/unit/database_tests.py

```diff
@@ -442,6 +442,17 @@ def test_all_docs_get(self):
         self.assertEqual(data['rows'][1]['key'], 'julia011')
         self.assertEqual(data['rows'][2]['key'], 'julia012')
 
+    def test_all_docs_get_with_long_type(self):
+        """
+        Test the all_docs GET request functionality
+        """
+        self.populate_db_with_documents()
+        data = self.db.all_docs(limit=10000000000, skip=10)
```

Comment:
yeah, needs to bigger than `sys.maxint` to yield a long int, or add the `L` suffix.